.github/workflows/pre-commit.yaml

```diff
@@ -3,7 +3,7 @@ name: pre-commit
 on:
   pull_request:
   push:
-    branches: [master]
+    branches: [$default-branch]
```

Comment:
I actually found it in github's starter workflow examples: https://github.com/actions/starter-workflows/blob/main/ci/django.yml

It seems like it's a [feature for workflow templates](https://github.com/actions/starter-workflows/pull/590#issuecomment-705313399) rather than workflows, so I'm actually not sure why it worked here. Anyway, since it's not documented for regular workflows, we should probably change it back to an explicit "master"  - WDYT?

Reply:
Either way is fine with me, though probably safer just to use `master`. Up to you.

Reply:
Yeah, I ended up putting master back in :+1: thanks for pointing this out!